iceoryx2/src/service/static_config/request_response.rs

```diff
@@ -130,6 +150,12 @@ impl StaticConfig {
         self.enable_safe_overflow_for_responses
     }
 
+    /// Returns true if fire and forget [`RequestMut`](crate::request_mut::RequestMut)s can be
+    /// sent from the [`Client`](crate::port::client::Client), otherwise false.
+    pub fn enable_fire_and_forget_requests(&self) -> bool {
```

Comment:
As this method doesn't enable fire and forget but returns if it is enabled, shouldn't the method be renamed to `is_fire_and_forget_enabled`?

Reply:
I renamed it to `does_support_fire_and_forget_requests()`.